lib/logflare/logs/vercel/nimble_vercel_parser.ex

```diff
@@ -261,29 +311,33 @@ defmodule Logflare.Logs.Vercel.NimbleLambdaMessageParser do
   report =
     ignore(string("REPORT RequestId: "))
     |> ignore(uuid)
-    |> ignore(tab)
+    |> ignore(choice([tab, string(" ")]))
     |> concat(
       ignore(string("Duration: "))
       |> concat(number_string)
-      |> ignore(string(" ms\t"))
+      |> ignore(string(" ms"))
+      |> ignore(optional(whitespace))
```

Comment:
```suggestion
      |> ignore(whitespace)
      |> ignore(string("ms"))
      |> ignore(optional(whitespace))
```

Wouldn't that work better?